tests/SlimFaas.Tests/ReplicasSynchronizationWorkerShould.cs

```diff
@@ -1,22 +1,24 @@
 ﻿using System.Collections;
 using Microsoft.Extensions.Logging;
 using Moq;
+using SlimFaas.Kubernetes;
 
 namespace SlimFaas.Tests;
 
 public class DeploymentsTestData:IEnumerable<object[]>
```

Comment:
another option to get strongly types is to inherit from TheoryData<T>. Then in the constructor, just call Add() to add tests iterations. This allow to remove the noise of object[].